packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -736,6 +747,17 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
         return true;
       }
     }
+
+    final File cachedFlavorFile = globals.fs.file(
+      globals.fs.path.join('build', 'test_cache', 'flavor.txt'),
+    );
+    final String? cachedFlavor = cachedFlavorFile.existsSync()
```

Comment:
Same here

Reply:
Addressed in [use sync file ops](https://github.com/flutter/flutter/pull/150461/commits/a3273e0f8ef8495d93f9a31b4a156471e87769be)